src/test/java/duke/DukeTest.java

```diff
@@ -0,0 +1,34 @@
+package duke;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class DukeTest {
+
+    @Test
+    public void testTodo() {
```

Comment:
If you were to choose one of the Task types to test, might be better to test Deadline or Event, as they are basically a more complex version of ToDo with the additional parameter of Date & Time that might give rise to errors.